build.ts

```diff
@@ -91,7 +91,7 @@ export const build = async (
               }
             : undefined,
         define: {
-          'process.env.version': JSON.stringify(
+          'process.env.VERSION': JSON.stringify(
```

Comment:
why the casing change?

Reply:
best practice for env constants to prevent accidental mis-use